### PR TITLE
slab detachment benchmark

### PR DESCRIPTION
**Pull request by cedrict**
This is the setup and manual entry for the slab detachment benchmark that A. Glerum presented in Glerum et al 2018.

### CODE REVIEW

File: doc/manual/manual.bib [hunk 1]
```diff
@@ -11490,4 +11490,67 @@ @book{jahu17
 year={2017}
 }
 
+@article{schm11,
+ title={{A simple analytical solution for slab detachment}},
+ author={S.M. Schmalholz},
+ journal=epsl,
+ volume={304},
+ number={},
+ pages={45-54},
```
Comment (bangerth) [inline doc/manual/manual.bib]:
```suggestion
 pages={45--54},
```

File: doc/manual/manual.bib [hunk 2]
```diff
@@ -11490,4 +11490,67 @@ @book{jahu17
 year={2017}
 }
 
+@article{schm11,
+ title={{A simple analytical solution for slab detachment}},
+ author={S.M. Schmalholz},
```
Comment (bangerth) [inline doc/manual/manual.bib]:
```suggestion
 author={S. M. Schmalholz},
```

File: doc/manual/manual.bib [hunk 3]
```diff
@@ -11490,4 +11490,67 @@ @book{jahu17
...
+ journal=epsl,
+ volume={304},
+ number={},
+ pages={45-54},
+ year={2011}
+}
+
+@Article{hitg14,
+AUTHOR = {Hillebrand, B. and Thieulot, C. and Geenen, T. and van den Berg, A. P. and Spakman, W.},
+TITLE = {Using the level set method in geodynamical modeling of multi-material flows and Earth's free surface},
```
Comment (bangerth) [inline doc/manual/manual.bib]:
```suggestion
TITLE = {Using the level set method in geodynamical modeling of multi-material flows and {E}arth's free surface},
```

File: doc/manual/manual.tex [hunk 4]
```diff
@@ -9372,6 +9372,57 @@ \subsubsection{The Burstedde variable viscosity benchmark}
...
+Slab detachment (slab break-off) may occur in the final stages of subduction
+as a consequence of the combination of a buoyant crust and strong slab pull.
+It is often invoked to explain 
+geophysical and geological observations such as 
+tomographic images of slab remnants and
+exhumed ultra-high-pressure rocks \cite{wosp00,vaal11,garm18}.
+
+This benchmark is based on the setup by S. Schmalholtz \cite{schm11}, which was subsequently 
+run with \aspect{} by A. Glerum \cite{gltf18}.
+The computational domain is a $1000 \text{km}\times 660 \text{km}$ box.
```
Comment (bangerth) [inline doc/manual/manual.tex]: :-)

File: doc/manual/manual.tex [hunk 5]
```diff
@@ -9372,6 +9372,57 @@ \subsubsection{The Burstedde variable viscosity benchmark}
...
+No-slip boundary conditions are imposed on the sides of the system, while free-slip
+boundary conditions are imposed at the top and bottom.
+
+\begin{figure}
+\centering
+\includegraphics[width=0.6\linewidth]{cookbooks/benchmarks/slab_detachment/drawing.png}
+\caption{Slab detachment benchmark: initial geometry
```
Comment (bangerth) [inline doc/manual/manual.tex]:
```suggestion
\caption{Slab detachment benchmark: Initial geometry.
```

File: doc/manual/manual.tex [hunk 6]
```diff
@@ -9372,6 +9372,57 @@ \subsubsection{The Burstedde variable viscosity benchmark}
...
+
+\begin{figure}
+\centering
+\includegraphics[width=0.6\linewidth]{cookbooks/benchmarks/slab_detachment/drawing.png}
+\caption{Slab detachment benchmark: initial geometry 
+\label{fig:slab_detachment_setup}}
+\end{figure}
+
+Two materials are present in the domain: the lithosphere and the mantle as shown 
+in Fig. (\ref{fig:slab_detachment_setup}). The gravity acceleration
```
Comment (bangerth) [inline doc/manual/manual.tex]:
```suggestion
in Fig. \ref{fig:slab_detachment_setup}. The gravity acceleration 
```

File: doc/manual/manual.tex [hunk 7]
```diff
@@ -9372,6 +9372,57 @@ \subsubsection{The Burstedde variable viscosity benchmark}
...
+\caption{Slab detachment benchmark: initial geometry 
+\label{fig:slab_detachment_setup}}
+\end{figure}
+
+Two materials are present in the domain: the lithosphere and the mantle as shown 
+in Fig. (\ref{fig:slab_detachment_setup}). The gravity acceleration 
+is Earth-like with $g=9.81 \text{m/s}^2$.
+The overriding plate is $80\text{km}$ thick and is placed at the top of the domain. 
+The already subducted lithosphere extends vertically into the mantle for $250 \text{km}$. 
+This slab has a density $\rho_s=3300\text{kg/m}^3$ and is characterised by a power-law flow law so that
```
Comment (bangerth) [inline doc/manual/manual.tex]:
```suggestion
This slab has a density $\rho_s=3300\text{kg/m}^3$ and is characterized by a power-law flow law so that 
```

**Comment by cedrict** [comment]
I have (hopefully) addressed all comments and squashed.

**Comment by bangerth** [comment]
In this cookbook (and I think others of yours), you never say where the files that implement the benchmark can be found. Take a look at how the other sections do that and if you wanted to fix this up in a follow-up, I'd gladly accept this!

**Pull request merged**